noir-projects/aztec-nr/aztec/src/note/constants.nr

```diff
@@ -1,6 +1,6 @@
-global MAX_NOTE_FIELDS_LENGTH: u64 = 20;
```

Comment:
Did the type change here to be able to compute `NoteViewer` offset as:

`let offset = page_index * MAX_NOTES_PER_PAGE;`
without casting.